src/pickledb.rs

```diff
@@ -71,6 +72,34 @@ impl PickleDb {
         }
     }
 
+    /// Attempts to load a DB from a file, and if it fails to load, constructs a new `PickleDb` instance.
+    ///
+    /// # Arguments
+    ///
+    /// * `db_path` - a path where the DB will be stored
+    /// * `dump_policy` - an enum value that determines the policy of dumping DB changes into the file. Please see
+    ///    [PickleDb::load()](#method.load) to understand the different policy options
+    /// * `serialization_method` - the serialization method to use for storing the data to memory and file
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use pickledb::{PickleDb, PickleDbDumpPolicy, SerializationMethod};
+    ///
+    /// let mut db = PickleDb::load_or_new("example.db", PickleDbDumpPolicy::AutoDump, SerializationMethod::Json);
+    /// ```
+    ///
+    pub fn load_or_new<P: AsRef<Path> + Clone>(
```

Comment:
Shouldn't need the `Clone` trait bound, since both `PickleDb::new()` and `PickleDb::load()` only require `P: AsRef<Path>` . You can use `db_path` as a reference.